etc/schema.json

```diff
@@ -2563,6 +2563,14 @@
                                             "sa_life_type_raw": {
                                                 "type": "integer"
                                             }
+                                        },
+                                        "patternProperties": {
+                                            "^sa_life_duration_[0-9]+$": {
```

Comment:
I do not like that. Was this decided against just keeping the first value ?

Reply:
Yes.